WordPress/Sniffs/Utils/I18nTextDomainFixerSniff.php

```diff
@@ -314,7 +314,7 @@ public function process_token( $stackPtr ) {
 
 			if ( preg_match( '`^[a-z0-9-]+$`', $this->new_text_domain ) !== 1 ) {
 				$this->phpcsFile->addWarning(
-					'The text-domain should be a simple lowercase text string with words separated by dashes. "%s" appears invalid',
+					'The text domain should be a simple lowercase text string with words separated by dashes. "%s" appears invalid',
```

Comment:
I disagree - for British (US) English, it reads fine as it is.

Reply:
I'll defer to your wisdom, though I wasn't talking about the grammar side of it, but of the fact that the message says that _`text-domain` should be lowercase with words separated by dashes_, leaving it as `text-domain`, makes it a good example of what people should use.
Similar to using `snake_case` in an error message instead of `snake case`.

Reply:
Let's keep it as is (in the PR), since this is us talking to the reader. If it needs an example of what a lowercase string with dashes are, then add that afterwards. I don't think it's necessary - it's not a confusing set of criteria.